projects/plugins/boost/app/modules/optimizations/minify/Minify.php

```diff
@@ -2,16 +2,13 @@
 
 namespace Automattic\Jetpack_Boost\Modules\Optimizations\Minify;
 
-use Automattic\Jetpack_Boost\Contracts\Pluggable;
+class Minify {
+	private static $called = false; // This class is called multiple times, so we need to make sure we only run some things once.
 
-// Allow overriding WordPress globals, as that is necessary to taking over script output.
-// phpcs:disable WordPress.WP.GlobalVariablesOverride.Prohibited
-
-class Minify implements Pluggable {
 	// @todo - handle PHP constants.
```

Comment:
TODOs should be turned into tickets.

Reply:
This was actually done before, I forgot to remove the todo, thanks.